app/controllers/projects_controller.rb

```diff
@@ -321,6 +321,12 @@ def upload_metadata
     }
   end
 
+  def metadata_fields
+    project_ids = (params[:projectIds] || []).map(&:to_i)
+
+    render json: Project.where(id: project_ids).map(&:metadata_fields).flatten.map(&:field_info)
```

Comment:
you should still do current_power.projects, right?

Reply:
Yes, thanks!